R/epi_keys.R

```diff
@@ -0,0 +1,42 @@
+#' Grab any keys associated to an epi_df
+#'
+#' @param x a data.frame, tibble, or epi_df
+#' @param ... additional arguments passed on to methods
+#'
+#' @return If an `epi_df`, this returns all "keys". Otherwise `NULL`
+#' @keywords internal
+#' @export
+epi_keys <- function(x, ...) {
```

Comment:
The term `epi_keys` seems to be [unintuitive](https://delphi-org.slack.com/archives/C028AUR6MV1/p1705521596770669) and [used in multiple ways in epipredict](https://delphi-org.slack.com/archives/C028AUR6MV1/p1696467136497469); and `ek` to refer to multiple things below.

Suggest:
- rename the `epi_keys` function `key_colnames` or maybe `key_names`

and maybe
- rename `ek` after killing time values to something like `geo_other_keynames`
- maybe rename `mv` and `ek` before killing time values to something more descriptive

Reply:
If you'd like, I could try to do this renaming as well.